### PR TITLE
Fix clang warnings in Calibration

### PR DESCRIPTION
**Pull request by mrodozov**

### CODE REVIEW

**Comment by mrodozov** [comment]
please test

File: Calibration/HcalCalibAlgos/test/HcalCorrPFCalculation.cc [hunk 1]
```diff
@@ -371,8 +371,7 @@ void HcalCorrPFCalculation::analyze(edm::Event const& ev, edm::EventSetup const&
       /*  Finding the closest cell at Hcal  */
       Int_t iphitrue = -10;
       Int_t ietatrue = 100;
-      HcalDetId tempId, tempId1, tempId2;
-      
+      HcalDetId tempId;
```
Comment (pohsun) [inline Calibration/HcalCalibAlgos/test/HcalCorrPFCalculation.cc]: It's better to remove also L388~L395, which are related to `tempId1` and `tempId2`.

**Review by pohsun — CHANGES_REQUESTED** [review]

**Comment by mrodozov** [comment]
please test

**Comment by pohsun** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**